html/partial/faq.mustache

```diff
@@ -8,8 +8,7 @@
 	</summary>
 	<div class="oi-faq__details">
 		{{#body}}
-			<p>{{.}}</p>
+			<p>{{{.}}}</p>
```

Comment:
Is this intentional? Probably is, just checking.

Reply:
Yeah, we put a link in one of the FAQs now, so I'm allowing markup to be rendered with triple braces.